### PR TITLE
Proposal: merge with WebVR polyfill in order to support Cardboard, and more devices sans HMD

### PR DESCRIPTION
**Pull request by borismus**

### CODE REVIEW

**Comment by mrdoob** [comment]
All this looks good to me! @jcarpenter what do you think?

File: index.html [hunk 1]
```diff
@@ -42,6 +41,7 @@
 		var sky, water;
 		var cameraPath;
 		var dolly;
+    var manager;
```
Comment (mrdoob) [inline index.html:44]: Please, notice that this file uses tabs for indentation.

**Comment by caseyyee** [comment]
Thanks @borismus.  Certainly makes juggling those effects and controls easier!

**Comment by borismus** [comment]
Thanks for the merge, Casey!

**Pull request merged**